packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -8,6 +9,8 @@ function nameWithoutNamespace(modelId: string): string {
 	return splitted.length === 1 ? splitted[0] : splitted[1];
 }
 
+const escapeQuotes = (str: string | undefined): string | undefined => str?.replace(/"/g, '\\"');
```

Comment:
```suggestion
const escapeQuotes = (str: string): string => str.replace(/"/g, '\\"');
```
I think it would be cleaner if the function signature is strictly `string` inout & `string` output